pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -491,6 +492,14 @@ func (c *Controller) preparePacket(tf *crdv1beta1.Traceflow, intf *interfacestor
 		} else if packet.DestinationIP.To4() != nil {
 			return nil, errors.New("destination Service does not have an IPv6 ClusterIP")
 		}
+		switch dstSvc.Spec.Ports[0].Protocol {
+		case corev1.ProtocolTCP:
+			packet.IPProto = protocol.Type_TCP
+			packet.TCPFlags = uint8(2)
+		case corev1.ProtocolUDP:
+			packet.IPProto = protocol.Type_UDP
```

Comment:
can we support SCTP as well here?

Reply:
We can add here but my view is that we should add here `SCTP` support when we decide to comprehensively support `SCTP` in traceflow, that means allow user to provide SCTP port/protocol in `tf.Spec.Packet.TransportHeader` as well. Also make necessary adjustments in `antctl traceflow` cmd.

Reply:
Sounds good, can you open an issue to add SCTP support?

Reply:
https://github.com/antrea-io/antrea/issues/6606